compiler/package.json

```diff
@@ -4,7 +4,7 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "node server.js"
+    "clean-ajs": "rm -r ./builds/analytics*.js"
```

Comment:
You can remove the `-r` flag (to avoid deleting folders) and include `--interactive=once` to ask for confirmation to the user (just in case, to avoid big mistakes.

Reply:
Good suggestion! I think `interactive=once` may be Linux-specific? I only see the `i` option on Unix/bash.